CHANGELOG.md

```diff
@@ -13,6 +13,25 @@ and this project adheres to Rust's notion of
 ### Changed
 - `orchard::pczt::SignerError` has added variants:
   - `InvalidExternalSignature`
+- All error enums in this crate are now `#[non_exhaustive]`, to allow future
+  error variants to be added without a SemVer break:
```

Comment:
Note this is not all *errors*, as there are some zero-sized structs used as errors (e.g. `orchard::builder::OutputError`. Making those non-exhaustive would let us add fields to them, but not turn them into enums; we'd either need to make them one-variant errors right now, or just defer that until the first time we need to distinguish a second error variant.

Reply:
Hmmm... If they're intended to be errors, they should probably be enums rather than structs. I would tend to prefer that we do that all at once with this change.